mandatory/3-function-output.js

```diff
@@ -1,14 +1,17 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getNumber() {
+  // generates random floating numbers between zero and 10 but never 10
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
 function s(w1, w2) {
+  // stringing together two strings-no spaces- and returns a string
   return w1.concat(w2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
+  return firstWord + " ".concat(secondWord) + " ".concat(thirdWord);
```

Comment:
This works perfectly well, but I would advise you to use one method of concatenating. In this line, you use both the plus sign + and the ``concat`` method. Since both methods are the same, it is always advised to use the plus sign as it is easier to read. Can you please change this line of code to reflect this?

Reply:
Sure will do thanks .